circle.yml

```diff
@@ -133,7 +133,7 @@ jobs:
       GENERATOR: Ninja
       BUILD_PARALLEL_JOBS: 8
       TEST_PARALLEL_JOBS: 8
-      CMAKE_OPTIONS: -DALETH_INTERPRETER_SHARED=ON
+      CMAKE_OPTIONS: -DALETH_INTERPRETER_SHARED=ON -DVMTRACE=ON
```

Comment:
why?

Reply:
It's useful feature. If you download binaries you might want to use it. Otherwise you get error that you have to build project with this option.

I'd like to make it an runtime option only. We have to review what it changes.